.travis.yml

```diff
@@ -1,36 +1,42 @@
 dist: xenial
 env:
   global:
-    - CC_TEST_REPORTER_ID=$TEST_REPORTER_ID
+  - CC_TEST_REPORTER_ID=$TEST_REPORTER_ID
 language: python
 os:
-  - linux
+- linux
 python:
-  - "3.7"
+- '3.7'
 
 install:
-  - pip install -U pip setuptools
-  - pip install pipenv
-  - pipenv install --dev
-  - pipenv run python setup.py develop
+- pip install -U pip setuptools
+- pip install pipenv
+- pipenv install --dev
+- pipenv run python setup.py develop
 
 before_script:
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-  - chmod +x ./cc-test-reporter
-  - ./cc-test-reporter before-build
+- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+- chmod +x ./cc-test-reporter
+- "./cc-test-reporter before-build"
 
 script:
-  - tox -e lint
-  - tox -e isort
-  - tox -e mypy
-  - tox -e tests
+- tox -e lint
+- tox -e isort
+- tox -e mypy
+- tox -e tests
 
 after_script:
-  - pipenv run coverage xml -o coverage.xml
-  - ./cc-test-reporter after-build --coverage-input-type coverage.py --exit-code $TRAVIS_TEST_RESULT
+- pipenv run coverage xml -o coverage.xml
+- "./cc-test-reporter after-build --coverage-input-type coverage.py --exit-code $TRAVIS_TEST_RESULT"
 
 deploy:
-  - provider: script
-    script: ci-scripts/semantic-release.sh
-    on:
-      branch: master
+- provider: script
+  script: ci-scripts/semantic-release.sh
+  on:
+    branch: master
+
+notifications:
+  slack:
+    secure: u0VC5i9ja+4ib45MSyEsD0zcuWP/LOezTar1kvVQdTOIlfFFLF+CmXOY6Slxzk8SScBTv0+Lq7VgegLGnLWVWoBTZDhO8eM5H6e6StN7OpRVAMHuzbIOzMo0IyrNekSWxF5volDrnVH+3Ubi/S/v7kblmSKlwoWSb+H05Ft7XCWvKEuBYtomTpUfsAMQmORcIvm3cGQeBrcpanjYA4s3xE06RLVbbI2QStzobgH8awWb2fSx7zHXCVUb4wsQaIXCcrDGJ//5VqEYWB8gPIY5bD8QXewkRYPMJG6BILa4vBcc615SvUEje1UfaabJrXggGbV2FBKmQnEB9n0/9EaFWly1OR0yuOND2s7bsuFvAHBwaqZ/ZaW27WTtKQp5fmoSnOM8PoMI2BHqSijvqmkN0OKiFErmGaX1V9xppogiXCzQzurGdGm7TpbTwRHOHriSMF4n6B6CfblzMbaOwICnDq0QSJ/huMop8e36bzE3AzdpVaUHBjz8znfNnMEQUihO8SCm81PWEC235K1HsAS9YLnmRp66lJ0vpDgp4Bdd8gqxXeQl7AOgkYVXFzmWaDGcNsUldqK796SSf0Icoj4pzUn2cIvt7n19B2x8nIhWbc8+xt+JSvZcaSNzFOeeg2VI+h2KRt3cQE2BPv6x5BZFaMDQph5BPXqxXQI9tHAJdLc=
```

Comment:
don't you want this in a travis env var?

Reply:
done. thx